benchmarks.cc

```diff
@@ -338,6 +369,7 @@ void run_throughput_benchmarks(HugePages& hp, std::vector<CpuTopologyInfo> const
 template<class Queue, bool Sender>
 void ping_pong_thread_impl(Queue* q1, Queue* q2, unsigned N, uint64_t* time) {
     uint64_t t0 = __builtin_ia32_rdtsc();
+    typename Reclaimer::region_guard guard{};
```

Comment:
This guard should be queue-specific and for other queue types it should resolve to an empty structure.